packages/kernel-6.12/kernel-6.12.spec

```
@@ -0,0 +1,1392 @@
+%global debug_package %{nil}
```

If you don't disable strip like this, then module signatures end up stripped and modules can't be loaded at runtime.

```suggestion
%global debug_package %{nil}
%global __strip /bin/true
```